ui/apps/platform/src/Containers/NetworkGraph/deployment/DeploymentDetails.tsx

```diff
@@ -17,7 +17,7 @@ import {
     TextContent,
     TextVariants,
 } from '@patternfly/react-core';
-import { ExclamationCircleIcon } from '@patternfly/react-icons';
+import InfoCircleIcon from '@patternfly/react-icons/dist/esm/icons/info-circle-icon';
```

Comment:
Is this linking to the single icon file (lodash-style) the recommended way to do it? 
(Mainly want to make sure PF team guarantees that the path should never change.)

Reply:
Yea basically. Seems like the PF examples do something similar too. That's a good question. Maybe I'll keep the same style used in the line above for now